.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +1,27 @@
 <!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
+HELP US HELP YOU BY PROVIDING AS MUCH OF THE FOLLOWING INFORMATION AS POSSIBLE AND
+AVOID DUPLICATES BY SEARCHING BEFORE CREATING A NEW ISSUE.
 -->
 
-# Description/Steps to reproduce
+## I'm submitting a ...
+<!-- Check one of the following options with a "x" -->
+<pre><code>
+[ ] Regression (a behavior that used to work and stopped working in a new release)
+[ ] Bug report
+[ ] Feature request <!-- Include a problem statement, use case scenario, example, code samples, etc. -->
+[ ] Documentation issue or request
+[ ] Support / Question
+[ ] Feedback / Comment / Thanks / Other
+</code></pre>
 
-<!--
-If feature: A description of the feature
-If bug: Steps to reproduce
--->
-
-# Link to reproduction sandbox
-
-<!--
-Link to an app sandbox for reproduction
-
-Note: Failure to provide a sandbox application for reproduction purposes will result in the issue being closed.
--->
-
-# Expected result
-
-<!--
-Also include actual results if bug
--->
+## Current Behaviour
+<!-- Include link to a sandbox application for reproduction is available -->
```

Comment:
> Include link to a sandbox application for reproduction is available

This does not read as an English sentence to me - I see two verbs "include" and "is available" (_Include link (...) is available_).

Few proposals for inspiration:

- Include a link to a sandbox application which we can use to reproduce the problem.
- Include a link to a minimal application reproducing the problem. We cannot help you if we don't know how to reproduce the problem you are experiencing!

Reply:
I meant to say `if available`. That said I'll update this to something along the suggestions you have since those are better.